breeze

```diff
@@ -458,35 +475,47 @@ do
       echo "Python version: ${PYTHON_VERSION}"
       echo
       shift 2 ;;
-    -E|--env)
-      export ENV="${2}";
-      echo
-      echo "Environment: ${ENV}"
-      echo
-      shift 2 ;;
     -B|--backend)
       export BACKEND="${2}";
       echo
       echo "Backend: ${BACKEND}"
       echo
       shift 2 ;;
-    -K|--kubernetes-version)
-      export KUBERNETES_VERSION="${2}";
+    -K|--start-kubernetes-cluster)
+      export START_KUBERNETES_CLUSTER="true";
+      export AIRFLOW_CONTAINER_FORCE_DOCKER_BUILD="true"
       echo
-      echo "Kubernetes version: ${KUBERNETES_VERSION}"
+      echo "Starting kubernetes cluster"
       echo
-      shift 2 ;;
+      shift ;;
+    -z|--recreate-kubernetes-cluster)
+      export RECREATE_KUBERNETES_CLUSTER="true";
+      echo
+      echo "Recreate kubernetes cluster"
+      echo
+      shift ;;
     -M|--kubernetes-mode)
       export KUBERNETES_MODE="${2}";
       echo
       echo "Kubernetes mode: ${KUBERNETES_MODE}"
       echo
       shift 2 ;;
+    -V|--kubernetes-version)
+      export KUBERNETES_VERSION="${2}";
+      echo
+      echo "Kubernetes version: ${KUBERNETES_VERSION}"
+      echo
+      shift 2 ;;
     -s|--skip-mounting-local-sources)
       SKIP_MOUNTING_LOCAL_SOURCES="true"
       echo "Skip mounting local sources: ${SKIP_MOUNTING_LOCAL_SOURCES}"
       echo
       shift ;;
+    -N|--no-deps)
+      export AIRFLOW_CONTAINER_DEPS="false"
+      echo "Only run airflow-testing image without dependent images"
```

Comment:
```diff
+     echo
      echo "Only run airflow-testing image without dependent images"
```
For consistency. BTW. Do we need so many enterów? Maybe two at the beginning and end of the getop loop are enough

Reply:
It's just more readable this way. I like space.

Reply:
But I will remove some.